docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,100 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. Based on requrested partition count, number of non polling static partitions are created to scan table portitions in parallel. Additional single partition is poller partition which scans for new data incrementally. Assumption is that there is an ordered column using which range queries can be formed. Only newly added data will be fetched by the polling. Also out of order insertion/deletion won't be supported.
```

Comment:
Strongly agree with Bhupesh's comments. We can assume reader is familiar with Java and perhaps basic Apex terms like tuple, operator, port and stream but all other terms should be explained in a way that such a reader would understand, before being used. If a term is discussed in greater detail further down in the same document, use a phrase like "see section titled ??? below".